spec/unit/resource_collection_spec.rb

```diff
@@ -151,6 +151,21 @@
         expect(collection[name]).to eq renamed_resource
       end
     end
+
+    context "with a resource and a renamed resource added in disorder" do
+      let(:resource) { ActiveAdmin::Resource.new namespace, resource_class }
+      let(:renamed_resource) { ActiveAdmin::Resource.new namespace, resource_class, as: name }
+      let(:name) { 'Administrators' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.